### ISSUE TITLE
can't Sync domain in v0.10.1

### ISSUE THREAD

**Issue by jsjcjsjc** [rrpproxy/whmcs-rrpproxy-registrar#63]
I update my  whmcs-rrpproxy-registrar from v0.8.4 to v0.10.1,
then i can't Sync domain
error message `SQLSTATE[42S22]: Column not found: 1054 Unknown column 'z.id_protection' in 'field list' (SQL: update `tbldomainpricing` as `p` inner join `mod_rrpproxy_zones` as `z` on CONCAT(".", z.zone) = `p`.`extension` set `p`.`idprotection` = `z.id_protection`, `p`.`eppcode` = `z.epp_required` where `p`.`autoreg` = keysystems)`

**Expected behavior**
System Settings->Domain Pricing->Click here to sync now->try to Sync domain

**Screenshots**
https://s6.jpg.cm/2021/11/27/LL0kcr.jpg

**Context (please complete the following information):**
 - WHMCS Version [8.13
 - PHP Version [7.4]

**Comment by papakai** [comment]
Hi @jsjcjsjc,

the easiest way is to drop table "mod_rrpproxy_zones" which is a custom table of this registrar module.
After having this done, navigate to Configurations > System Settings > Domain Registrars and click on the Configure button of this module. The table will then be recreated from scratch.

We introduced a new feature (including sync of id protection in Registrar TLD Sync on demand) for which we had to introduce this new column. This column is missing in your case therefore... A missing auto-upgrade step.

Sorry for the inconveniences caused.
HTH, if not let us know.

@h9k fyi.

Kai
